iModelCore/ECDb/ECDb/ECSql/PragmaECSqlPreparedStatement.cpp

```diff
@@ -1021,13 +1021,13 @@ double PragmaVal::GetDouble() const {
 //---------------------------------------------------------------------------------------
 bool PragmaVal::GetBool() const {
     if(IsBool()) {
-        return std::any_cast<bool>(m_val);
+        return m_integer;
```

Comment:
Did you mean `m_bool`?

Reply:
Thanks for pointing out. I have fixed this one.